tracing/zipkin/http_test.go

```diff
@@ -0,0 +1,160 @@
+package zipkin_test
+
+import (
```

Comment:
can you group imports like this (with a newline between groupings):
- standard lib
- openzipkin
- gokit